conformance/lang/values-types-and-variables/array.balt

```diff
@@ -0,0 +1,99 @@
+Test-case: output
+Description: Test valid array types and values
+Label: array
+
+const LENGTH = 2;
```

Comment:
Let's not use const, in this test. This should be a different test.

Reply:
Shouldn't the tests for constant-reference-expr as the array-length be included in this file though?

If not here, it would probably have to go under constant-reference-expr right? Isn't this the better category?

Reply:
I think it being in this file is OK, but it should be under a different sections that has `Label: array-type-descriptor, constant-reference-expr `.